### PR TITLE
Replaced all reference to egirna/icap-client with integrationworks/ic…

### PR DESCRIPTION
**Pull request by tennengabr**
This change successfully prevents go mod tidy from importing the egirna/icap-client and instead uses the integrationworks/icap-client module. Also imports the egirna/icap v0.2.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # icap-client
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
-[![Project status](https://img.shields.io/badge/version-0.1.0-green.svg)](https://github.com/egirna/icap-client/releases)
-[![GoDoc](https://godoc.org/github.com/egirna/icap-client?status.svg)](https://godoc.org/github.com/egirna/icap-client)
+<!-- [![Project status](https://img.shields.io/badge/version-0.1.0-green.svg)](https://github.com/egirna/icap-client/releases) -->
```
Comment (vaughnbuttiw) [inline README.md:4]: I'm inclined to leave the doc links as is.

**Review by vaughnbuttiw — CHANGES_REQUESTED** [review]
I think you should revert the changes to README.md. Apart from that, all good.

File: README.md (hunk 1 above)
Comment (tennengabr) [reply README.md:4]: I think it is still there just commented out. Should I uncomment it?

File: README.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ By default the icap-client will dump the debugging logs to the standard output(s
   ic.SetDebugOutput(f)
 ```
 
-For more details, see the [docs](https://godoc.org/github.com/egirna/icap-client) and [examples](examples/).
+<!-- For more details, see the [docs](https://godoc.org/github.com/egirna/icap-client) and [examples](examples/). -->
```
Comment (vaughnbuttiw) [inline README.md:86]: This one should be reverted too.

**Review by vaughnbuttiw — APPROVED** [review]

**Comment by tennengabr** [comment]
Moving ticket in repo kong-plugin-antivirus-golang project board relating to this task to complete.

**Pull request merged**